app/components/profile_picture/image.tsx

```diff
@@ -77,6 +79,16 @@ const Image = ({author, forwardRef, iconSize, size, source, url}: Props) => {
 
         const pictureUrl = client.getProfilePictureUrl(author.id, lastPictureUpdate);
         const imgSource = source ?? {uri: `${serverUrl}${pictureUrl}`};
+        if (imgSource.uri?.startsWith('file://')) {
```

Comment:
are we sure the `file://` prefix is always applicable? should we handle also when the local file is `/some/path/to/file` ?

Reply:
The only place where we add the source is in `edit_profile_picture.tsx`, where we either prefix the uri with the server url or `file://`